.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "es2015"
+    "es2015-native-modules"
```

Comment:
I was concerned about the same thing (removing by index).  Glad we're in agreement.

I'll take care of this.